### ISSUE TITLE
sp_QuickieStore - Count of executions across all plan ids?

### ISSUE THREAD

**Issue by MinnRick** [erikdarlingdata/DarlingData#286]
Hi Erik.  Is there parameter setting or other tweak available in sp_QuickieStore that displays the total execution count for all plan ids for each query in the script's result set?  Counts per plan id is useful but it'd also he helpful to have a total across all plans.  Thanks.

**Comment by erikdarlingdata** [comment]
@MinnRick no, there's nothing like that. If you'd like to build it, I'll leave the PR open for you to work on it.
